pyproject.toml

```
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "antipasti"
+version = "0.0.1"
+description = "Finally, a collection of useful stuff that you implement in your project every bloody time"
+authors = [
+    "Oleh Chyhyryn <oleh.chyhyryn@outlook.com>",
+    "Dmytro Selishchev <zibertscrem@gmail.com>"
+]
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8.1"
+
+
+[tool.poetry.group.dev.dependencies]
+mypy = "^1.7.0"
+flake8 = "^6.1.0"
+flake8-pyproject = "^1.2.3"
+flake8-type-checking = "^2.5.1"
+black = "^23.11.0"
+isort = "^5.12.0"
+pytest = "^7.4.3"
+pyupgrade = "^3.15.0"
+commitizen = "^3.12.0"
```

For all those deps, I recommend to use `~` instead of `^` as version specifier cause it will not automatically upgrade minor versions

```suggestion
mypy = "~1.7.0"
flake8 = "~6.1.0"
flake8-pyproject = "~1.2.3"
flake8-type-checking = "~2.5.1"
black = "~23.11.0"
isort = "~5.12.0"
pytest = "~7.4.3"
pyupgrade = "~3.15.0"
commitizen = "~3.12.0"
```